### PR TITLE
Adding default box file and .gitkeep file to stub out modules dir.

### PR DESCRIPTION
**Pull request by sk33lz**

### CODE REVIEW

**Comment by sk33lz** [comment]
This PR gets us to a working VM that builds the ubuntu-14.04:intermediary image we need with working composer ENV PATH and working default ENV PATH.

File: puppet/manifests/main.pp [hunk 1]
```diff
@@ -40,3 +40,9 @@
 package { 'git':
   ensure => 'installed',
 }
+
+file { '/workspace':
```
Comment (lliss) [inline puppet/manifests/main.pp]: I think you don't need this since it will be handled by vagrant.

File: Vagrantfile [hunk 2]
```diff
@@ -4,8 +4,10 @@
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "lepew"
+  #config.vm.box = "lepew"
```
Comment (lliss) [inline Vagrantfile]: I prefer just dumping commented out junk.

File: Vagrantfile [hunk 3]
```diff
@@ -4,8 +4,10 @@
...
   #config.vm.box_url = "http://domain.com/path/to/above.box"
+  config.vm.box = "puppetlabs/ubuntu-14.04-64-puppet-1.0.0"
```
Comment (lliss) [inline Vagrantfile]: Do we need this older box? Can we just use the latest and drop the line about the box_url below.

Comment (sk33lz) [inline Vagrantfile]: The last thing I did was try updating the box number in both lines. Let me try it by updating in the `config.vm.box line` and remove the second line.

File: Vagrantfile (hunk 2 above)
Comment (sk33lz) [inline Vagrantfile]: Yeah, I meant to remove this after our discussion about the lepew box.

**Comment by sk33lz** [comment]
Cleaned this up and squashed it.

File: Vagrantfile [hunk 4]
```diff
@@ -4,8 +4,7 @@
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "lepew"
-  #config.vm.box_url = "http://domain.com/path/to/above.box"
+  config.vm.box = "puppetlabs/ubuntu-14.04-64-puppet-1.0.3"
```
Comment (lliss) [inline Vagrantfile]: I'm pretty sure you can set this to 
`puppetlabs/ubuntu-14.04-64` and it will always pull the latest.

Comment (sk33lz) [inline Vagrantfile]: Negative.

```
The box 'puppetlabs/ubuntu-14.04-64' could not be found or
could not be accessed in the remote catalog.
```

Comment (sk33lz) [inline Vagrantfile]: Let me try it with box `puppetlabs/ubuntu-14.04-64-puppet` for good measure.

Comment (sk33lz) [inline Vagrantfile]: `puppetlabs/ubuntu-14.04-64-puppet` seems to work, but I have no idea what version it's actually using. I'd rather stick with the `puppetlabs/ubuntu-14.04-64-puppet-1.0.3`, as I can actually see it's grabbing that version and I've tested that it works. I also don't think we will be missing out on any new releases as they haven't released a new box version in over a year.

Comment (sk33lz) [inline Vagrantfile]: I took a look at their site again to see when the date of the release was for 1.0.3 and they have redesigned their site and have some great documentation right on the version page now. This is what they have for the syntax.

```
Vagrant.configure("2") do |config|
  config.vm.box = "puppetlabs/ubuntu-14.04-64-puppet"
  config.vm.box_version = "1.0.3"
end
```

Comment (sk33lz) [inline Vagrantfile]: I'm going to change it up to this inside or config and do a provision locally.

```
config.vm.box = "puppetlabs/ubuntu-14.04-64-puppet"
config.vm.box_version = "1.0.3"
```

Comment (sk33lz) [inline Vagrantfile]: After some testing I verified that using just `config.vm.box = "puppetlabs/ubuntu-14.04-64-puppet"` will download the latest box. I figured this out by removing all the boxes on my local machine and then running `vagrant up` again with the changes.

**Comment by sk33lz** [comment]
Should be all set.

**Pull request merged**